ads/doubleclick.js

```diff
@@ -21,11 +21,27 @@ import {loadScript, checkData} from '../src/3p';
  * @param {!Object} data
  */
 export function doubleclick(global, data) {
+  const experimentFraction = 0.01;
+
   checkData(data, [
-    'slot', 'targeting', 'categoryExclusion',
+    'slot', 'targeting', 'categoryExclusions',
     'tagForChildDirectedTreatment', 'cookieOptions',
     'overrideWidth', 'overrideHeight',
   ]);
+
+  if (Math.random() < experimentFraction) {
```

Comment:
Maybe it makes sense to add some kind of opt-in based on the URL of the doc? like `?glade=1`?